# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/libcubeb/moz.yaml
# Commit: 5b05c441f46f
# Full Hash: 5b05c441f46f8a22043a8e8dc7163c77b8635953
# Author: Matthew Gregan <kinetik@flim.org>
# Date: 2022-01-08 09:40:24
# Regressor Bug: 1748948
# File Overlap Count: 2
# Description:
#   Bug 1748948 - Update cubeb to 3d598eff.  r=cubeb-reviewers,padenot
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D135306
# ==============================================================================

diff -r 57d47cc95fa3 -r 5b05c441f46f media/libcubeb/moz.yaml
--- a/media/libcubeb/moz.yaml	Fri Jan 07 21:37:10 2022 +0200
+++ b/media/libcubeb/moz.yaml	Fri Jan 07 19:20:31 2022 +0000
@@ -19,5 +19,5 @@
   license: "ISC"
 
   # update.sh will update this value
-  release: "773f16b7ea308392c05be3e290163d1f636e6024-dirty (2021-12-08 11:09:25 +1300)"
+  release: "3d598eff576ea5939b63119d54a8a54183eec144 (2022-01-07 14:34:59 +1300)"
 